python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py

```diff
@@ -250,19 +255,35 @@ def __init__(self, m1m3: MetaSAL):
                 self.TEXT_START,
                 self.TEXT_ENABLE,
                 self.TEXT_RAISE,
+                self.TEXT_PAUSE_RAISING,
                 self.TEXT_ENTER_ENGINEERING,
                 self.TEXT_STANDBY,
             ],
             {
                 self.TEXT_RAISE: "raiseM1M3",
+                self.TEXT_PAUSE_RAISING: "pauseM1M3RaisingLowering",
+                self.TEXT_RESUME_RAISING: "resumeM1M3RaisingLowering",
```

Comment:
"pauseM1M3RaisingLowering" -> "pauseM1M3Raising" ?
"resumeM1M3RaisingLowering" -> "resumeM1M3Raising" ?

Reply:
This is SAL/DDS command, so that cannot be changed.